docs/guides/on-demand.mdx

```diff
@@ -67,17 +67,17 @@ the response will return the `id` of that asset
 
 The response will indicate the status of the uploaded assets as status:
 
-- `waiting` or `status`: "ready".
+- `status: "waiting"` or `status: "ready"`.
 
-- If status: "waiting" is returned in the response, the asset is not available:
+- If `status: "waiting"` is returned in the response, the asset is not available:
 
-  - make the API call again until status: "ready".
+  - make the API call again until `status: "ready"`.
 
-- If the status indicates that it is "ready", it is available.
+- If the status indicates that it is `"ready"`, it is available.
 
 ## Listing the Tasks
 
-To check the status on an asset imported /import to Livepeer or exported /export to IPFS, get the `task.id` from the response of the uploaded asset.
+To check the status on an asset uploaded with URL `/import` to Livepeer or exported `/export` to IPFS, get the `task.id` from the response of the uploaded asset.
```

Comment:
Also, we should stop referencing the `/import` API if we are not calling it that anymore. I've just opened a pull request to register that API under `/upload/url` instead, WDYT? 
https://github.com/livepeer/studio/pull/1182

The only thing that might be confusing is that the "direct upload" (upload from local storage) APIs are served under `/upload/:url` instead, so that's one extra reason to stop "documenting" any format about the "upload from local storage" URLs (in fact, I think I'm gonna change it to `/uplaod/direct/:token` on that same PR instead).